deploy/job-manager/nginx.conf.ctmpl

```diff
@@ -17,8 +17,13 @@ http {
         root /ui/dist;
 
         # Set up HTTP Basic Auth for Nginx here:
-        auth_basic "Username and password required";
-        auth_basic_user_file /etc/apache2/.htpasswd;
+
+        {{ if env "USE_PROXY" | parseBool }}
+            auth_basic "Username and password required";
```

Comment:
It seems the indentation here will result in indentations in the rendered nginx.conf file, how about removing them?

```
{{ if env "USE_PROXY" | parseBool }}
        auth_basic "Username and password required";
        auth_basic_user_file /etc/apache2/.htpasswd;
        {{ else }}
        auth_basic off;
        {{ end }}
```